web_accounts_list.json

```diff
@@ -4263,6 +4263,62 @@
       "known_accounts" : ["summit1g","cohhcarnage"],
       "category" : "gaming",
       "valid" : false
-      }
+      },
+      {
+        "name" : "fansly",
+        "check_uri" : "https://apiv2.fansly.com/api/v1/account?usernames={account}",
+        "pretty_uri": "https://fansly.com/{account}/posts",
+        "account_existence_code" : "200",
+        "account_existence_string" : "username",
+        "account_missing_string" : "response: []",
+        "account_missing_code" : "200",
+        "known_accounts" : ["Mikomin","test"],
+        "category" : "XXXPORNXXX",
+        "valid" : true
+        },
+        {
+          "name" : "mym.fans",
+          "check_uri" : "https://mym.fans/{account}",
+          "account_existence_code" : "200",
+          "account_existence_string" : "posts",
+          "account_missing_string" : "Il n’y a rien ici…",

```

Comment:
I think this response is language specific. When I go to the site, my browser shows "There’s nothing here...". Can you pick a different existence_string that is not lang dependent?

Reply:
Have you tried to test this entry by running the script?